src/reduced_basis/rb_eim_construction.C

```diff
@@ -436,6 +475,43 @@ Real RBEIMConstruction::get_max_abs_value_in_training_set() const
   return _max_abs_value_in_training_set;
 }
 
+void RBEIMConstruction::store_eim_rhs_for_training_set()
+{
+  LOG_SCOPE("store_eim_rhs_for_training_set()", "RBEIMConstruction");
+
+  RBEIMEvaluation & eim_eval = get_rb_eim_evaluation();
+
+  std::vector<DenseVector<Number>> & eim_rhs_values = get_rb_eim_evaluation().eim_rhs_values;
+  eim_rhs_values.clear();
+  eim_rhs_values.resize(get_n_training_samples());
+  for (auto i : make_range(get_n_training_samples()))
+    {
+      // Make a copy of the parametrized function for training index, since we
+      // will modify this below to give us a new basis function.
+      auto local_pf = _local_parametrized_functions_for_training[i];
+
+      unsigned int RB_size = get_rb_eim_evaluation().get_n_basis_functions();
+      if (RB_size > 0)
+        {
+          // Get the right-hand side vector for the EIM approximation
+          // by sampling the parametrized function (stored in solution)
+          // at the interpolation points.
+          DenseVector<Number> EIM_rhs(RB_size);
+          for (unsigned int i=0; i<RB_size; i++)
```

Comment:
We've now got a -Wshadow warning (error, in my usual configuration) in libMesh master.

Man I hope Civet is up and running again soon.

Reply:
The index-change fix is simple enough *if* the code is currently correct, but I'll leave it to @jwpeterson and @dknez just in case there's an actual error being flagged by the warning.

Reply:
I'll push a fix for this shortly... looks to me like all the instances of `i` in the inner loop can be changed to `j` without issue, the outer `i` is only used for setting the `local_pf` reference.